.editorconfig

```diff
@@ -8,12 +8,20 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.{java,kt}]
+ij_kotlin_imports_layout = *
+
 # Disable wildcard imports
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
 ij_java_class_count_to_use_import_on_demand = 999
-# Enable trailing comma
-ktlint_disabled_rules=trailing-comma-on-call-site,trailing-comma-on-declaration-site
+
+ktlint_code_style = android_studio
+
+# Disable trailing comma
+ktlint_standard_trailing-comma-on-call-site = disabled
+ktlint_standard_trailing-comma-on-declaration-site = disabled
```

Comment:
```suggestion
ij_kotlin_allow_trailing_comma = false
ij_kotlin_allow_trailing_comma_on_call_site = false
```

Reply:
We can use these instead.

https://pinterest.github.io/ktlint/latest/rules/standard/#trailing-comma-on-declaration-site
https://pinterest.github.io/ktlint/latest/rules/standard/#trailing-comma-on-call-site